### PR TITLE
Fix "offset in function" for uprobes

### PR DESCRIPTION
**Pull request by florian-kuebler**
On uprobes, libunwindstack's address info does not contain a valid
"offset in function". Thus, we need to treat the whole uprobes map
as one function and compute the offset based on the map's start
and the pc.

This has been done previously, but was removed by #3616.

Test: Run test.

### CODE REVIEW

File: src/LinuxTracing/UprobesUnwindingVisitorTest.cpp [hunk 1]
```diff
@@ -1283,7 +1292,7 @@ TEST_F(UprobesUnwindingVisitorTest, VisitPatchableCallchainSampleSendsCompleteCa
 
   EXPECT_CALL(maps_, Find).WillRepeatedly(Return(&kTargetMapInfo));
   auto fake_patch_callchain = [](pid_t /*tid*/, uint64_t* callchain, uint64_t callchain_size,
-                                 orbit_linux_tracing::LibunwindstackMaps *
+                                 orbit_linux_tracing::LibunwindstackMaps*
```
Comment (dpallotti) [inline src/LinuxTracing/UprobesUnwindingVisitorTest.cpp:1295]: clang-format fails here and below. It happens to me every time with this file. Very annoying.

File: src/LinuxTracing/UprobesUnwindingVisitorTest.cpp [hunk 2]
```diff
@@ -738,14 +746,15 @@ TEST_F(UprobesUnwindingVisitorTest, VisitStackSampleWithinUprobeSendsInUprobesCa
...
   EXPECT_EQ(actual_callstack_sample.callstack().type(), orbit_grpc_protos::Callstack::kInUprobes);
   EXPECT_THAT(
       actual_address_infos,
       UnorderedElementsAre(
-          AllOf(Property(&orbit_grpc_protos::FullAddressInfo::absolute_address, kUprobesMapsStart),
+          AllOf(Property(&orbit_grpc_protos::FullAddressInfo::absolute_address,
+                         kUprobesMapsStart + 1),
                 Property(&orbit_grpc_protos::FullAddressInfo::function_name, kUprobesName),
-                Property(&orbit_grpc_protos::FullAddressInfo::offset_in_function, 0),
+                Property(&orbit_grpc_protos::FullAddressInfo::offset_in_function, 1),
```
Comment (dpallotti) [inline src/LinuxTracing/UprobesUnwindingVisitorTest.cpp:757]: Nice to test this!

**Review by dpallotti — APPROVED** [review]
Thanks

File: src/LinuxTracing/UprobesUnwindingVisitorTest.cpp (hunk 2 above)
Comment (florian-kuebler) [reply src/LinuxTracing/UprobesUnwindingVisitorTest.cpp:757]: yeah, my future self will be happy :)

**Pull request merged**